cfme/modeling/base.py

```diff
@@ -48,12 +48,11 @@ def collections(self):
 
     def __dir__(self):
         internal_dir = dir(super(EntityCollections, self))
-        return internal_dir + self._availiable_collections.keys()
+        return internal_dir + list(self._availiable_collections)
```

Comment:
**REQUIRED:** Though this may be very pythonic. I'm actually thinking we either need a comment here stating that ```self._available_collections``` is a dict or we need to do the slightly less efficient ```list(self._available_collections.keys())```